src/rest-server/src/middlewares/v2/protocol.js

```diff
@@ -70,11 +73,16 @@ const convertPriority = (priorityClass) => {
 };
 
 const hivedValidate = (protocolObj) => {
+  if (!hivedSchema.validate(protocolObj)) {
```

Comment:
could you add another hived middleware file and import here?

Reply:
move hived validate to seprate file